onward/test/ContentCardControllerTest.scala

```diff
@@ -1,11 +1,10 @@
 package test
-
-import controllers.RichLinkController
+import controllers.ContentCardController
 import org.scalatest.{BeforeAndAfterAll, DoNotDiscover, FlatSpec, Matchers}
-import play.api.test._
 import play.api.test.Helpers._
+import play.api.test._
 
-@DoNotDiscover class RichLinkControllerTest
+@DoNotDiscover class ContentCardControllerTest
```

Comment:
I suppose this may be consolidated with the `RichLinkControllerTest` if the controllers could be deduped a bit, but the tests look great to me!